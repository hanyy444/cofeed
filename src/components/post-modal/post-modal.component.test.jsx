import {render, screen} from "@testing-library/react"
import PostModal from "./post-modal.component"

describe('Test post-modal component', () => { 
    it('Should render without crashing', () => {
        expect.assertions(2)
        render(<PostModal/>)
        const postModal = screen.getByTestId('post-modal')
        expect(postModal).toBeInTheDocument()
        expect(postModal).toMatchSnapshot()
    })
})