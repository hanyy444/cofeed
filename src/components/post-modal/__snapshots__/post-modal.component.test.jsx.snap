// Vitest Snapshot v1

exports[`Test post-modal component > Should render without crashing 1`] = `
<div
  class="post-modal"
  data-testid="post-modal"
  id="post-modal"
>
  <div
    class="post-modal__content"
  />
</div>
`;
